lib/service/database/table/announcement_table.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:logger/logger.dart';
+import 'package:sqflite/sqflite.dart';
+
+import '../table.dart';
+
+class AnnouncementTable extends Table {
+  final String titleCol;
+  final String descriptionCol;
+  final String categoryCol;
+
+  const AnnouncementTable._internal()
+      : titleCol = "title",
+        descriptionCol = "description",
+        categoryCol = "category",
+        super(
+          tableName: "announcement",
+        );
+
+  static AnnouncementTable instance = const AnnouncementTable._internal();
+
+  @override
```

Comment:
@irfanSilkron explain kenapa pakai override

Reply:
Sebab AnnouncementTable inherit from Table class & dua2 class ada method 'getCreateTableQuery' . So guna override utk replace the method from its parent class (Table).